src/Redux/Book/reducer.js

```diff
@@ -33,4 +34,45 @@ const bookReducer = (state = initialState, action) => {
   }
 };
 
+export const getBooksList = () => async (dispatch) => {
+  const booksList = await fetch(`${baseURL}`)
+    .then((response) => response.json());
+  const booksID = Object.keys(booksList);
+  const formatedBooks = [];
+  booksID.map((key) => formatedBooks.push({
+    id: key,
+    title: booksList[key][0].title,
+    category: booksList[key][0].category,
+  }));
+  dispatch(addBook(formatedBooks));
+};
+
+export const postBook = (newBook) => async (dispatch) => {
+  await fetch(`${baseURL}`, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: newBook.id,
+      title: newBook.title,
+      category: newBook.category,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  dispatch(addBook(newBook));
+};
+/*
+  deleteBook = (id) => async (dispatch) => {
+  await fetch(`${baseURL}/${id}`, {
+    method: 'DELETE',
+    body: JSON.stringify({
+      item_id: id,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  dispatch(removeBook(id));
+}; */
+
```

Comment:
Please remove any unused commented code in respect of the YAGNI principle. It is best practice to keep your code clean and readable.